lib/Limitation/Mapper/ContentTypeLimitationMapper.php

```diff
@@ -33,4 +33,9 @@ protected function getSelectionChoices()
 
         return $contentTypeChoices;
     }
+
+    protected function getChoiceFieldOptions()
+    {
+        return ['required' => false];
```

Comment:
If we have to add `['required' => false]` everywhere, maybe we should add this as default in abstract mapper